tests/std/tests/P1135R6_atomic_wait/test.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _ENABLE_ATOMIC_ALIGNMENT_FIX
+
+#include <atomic>
+#include <cassert>
+#include <chrono>
+#include <thread>
+
+template <class UnderlyingType>
+void test_atomic_wait_func(
+    UnderlyingType old_value, UnderlyingType new_value, std::chrono::steady_clock::duration waiting_duration) {
+
+    constexpr std::ptrdiff_t seq_max_size = 10;

```

Comment:
Technically need to include `<cstddef>` if you want `std::ptrdiff_t`.

Reply:
Let it be `int` then